docs/source/tutorials/llama_kd_tutorial.rst

```diff
@@ -0,0 +1,228 @@
+
+Using a fine-tuned teacher model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The default settings in the config uses the fine-tuned teacher model. Now, let’s take a look at the
+effects of not fine-tuning the teacher model first. To change the teacher model, you can modify the
+teacher_checkpointer in the config:
+
+.. code-block:: yaml
+
+  teacher_checkpointer:
+    _component_: torchtune.training.FullModelHFCheckpointer
+    checkpoint_dir: /tmp/Meta-Llama-3.1-8B-Instruct/
+    checkpoint_files: [
+        model-00001-of-00004.safetensors,
+        model-00002-of-00004.safetensors,
+        model-00003-of-00004.safetensors,
+        model-00004-of-00004.safetensors
+    ]
+
+In the table below, we can see that standard fine-tuning of the 1B model achieves better accuracy
+than the baseline 1B model. By using the fine-tuned 8B teacher model, we see comparable results
+for truthfulqa and improvement for hellaswag and commonsense. When using the baseline 8B as a
+teacher, we see improvement across all metrics, but lower than the other configurations.
+
+.. image:: /_static/img/kd-finetune-teacher.png
+
+Taking a look at the losses, using the baseline 8B as teacher results in a higher loss than
+using the fine-tuned teacher model. The KD loss also remains relatively constant, suggesting
+that the teacher model should have the same distributions as the transfer dataset.
+
+Using a fine-tuned student model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For these experiments, let's take a look at the effects of KD when the student model is already
+fine-tuned. In these experiments, we look at different combinations of baseline and fine-tuned 8B
+and 1B models. To change the student model, you can first fine-tune the 1B model then modify the
+student model checkpointer in the config:
+
+.. code-block:: yaml
+
+ checkpointer:
+    _component_: torchtune.training.FullModelHFCheckpointer
+    checkpoint_dir: /tmp/Llama-3.2-1B-Instruct/
+    checkpoint_files: [
+      hf_model_0001_0.pt
+    ]
+
+Using the fine-tuned student model boosts accuracy even further for truthfulqa, but the accuracy
+drops for hellaswag and commonsense. Using a fine-tuned teacher model and baseline student
+model achieved the best results on hellaswag and commonsense dataset. Based on these findings,
+the best configuration will change depending on which evaluation dataset and metric you are optimizing for.
+
+.. image:: /_static/img/kd-finetune-student.png
+
+Based on the loss graphs, using a fine-tuned teacher model results in a lower loss irrespective of
+whether the student model is fine-tuned or not. It's also interesting to note that the class loss
+starts to increase when using a fine-tuned student model.
+
+Hyperparameter tuning: learning rate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, the config has the learning rate as 3e-4, same as LoRA configs. For these experiments,
+we changed the learning rate from as high as 1e-3 to as low as 1e-5. To change the learning rate,
```

Comment:
mega nit: should be able to use latexify this using something like :math:`1e^{-1}`

Reply:
Changed all to use math.